app/views/globalPromotions.html

```diff
@@ -77,11 +78,21 @@ <h2 translate>{{$ctrl.getMinistryName()}} Promotions</h2>
             </tr>
           </thead>
           <tbody>
-            <tr ng-repeat="promotion in $ctrl.promotions">
+            <tr
+              ng-repeat="promotion in $ctrl.promotions"
+              ng-init="isActive = $ctrl.isActive(promotion)"
+            >
               <td>
                 <strong>{{promotion.code}}</strong>
               </td>
               <td>{{promotion.amount | localizedCurrency:'USD'}}</td>
+              <td class="text-center">
+                <span
+                  ng-class="isActive ? 'label label-success status-label' : 'label label-danger status-label'"
```

Comment:
You have to repeat `label` and `status-label` classes. Does `class="label status-label" ng-class="isActive 'label-success' ? 'label-danger'"` work?